client/src/app/tabs/cloud-dmn/DmnEditor.js

```diff
@@ -91,6 +94,40 @@ export class DmnEditor extends CachedComponent {
     this.propertiesPanelRef = React.createRef();
 
     this.handleResize = debounce(this.handleResize);
+
+    this.engineProfile = new EngineProfileHelper({
+      get: () => {
+        const modeler = this.getModeler();
+        const activeViewer = modeler.getActiveViewer();
+
+        if (!activeViewer) {
```

Comment:
When would this happen? If it can happen, why not test it?

Reply:
It occurred that we can safely remove this.